geometry_msgs/Pose pose_above
geometry_msgs/Pose pose_grip
bool gripper_close
---
bool success
